tests/specs/install/jsr_exports_uses_locked_multiple_in_lockfile/deno.lock

```diff
@@ -0,0 +1,36 @@
+{
+  "version": "5",
+  "specifiers": {
+    "jsr:@denotest/add@1": "1.0.0",
+    "jsr:@denotest/multiple-exports@0.7": "0.7.0",
+    "jsr:@denotest/multiple-exports@0.7.1": "0.7.1",
```

Comment:
The case that broke with https://github.com/denoland/deno/pull/29642.

We would get the exports for 0.7.0, but then when building the graph we would end up resolving `0.7` -> `0.7.1`, making the exports invalid.